apps/tangle-dapp/containers/ApiDevStatsContainer/ApiDevStatsContainer.tsx

```diff
@@ -0,0 +1,151 @@
+'use client';
+
+import { Expand } from '@webb-tools/icons';
+import { SkeletonLoader, Typography } from '@webb-tools/webb-ui-components';
+import { FC, useEffect, useState } from 'react';
+
+import usePromise from '../../hooks/usePromise';
+import { getPolkadotApiPromise, getPolkadotApiRx } from '../../utils/polkadot';
+
+const MEGABYTE_FACTOR = 0.000001;
```

Comment:
can you add comments to explain the `MEGABYTE_FACTOR` variable and the `formatBytes` function here